examples/module/dynamics/tank_robot.py

```diff
@@ -0,0 +1,115 @@
+import math
+import pypose as pp
+import torch as torch
+import matplotlib.pyplot as plt
+from matplotlib.patches import Ellipse
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+
+class TankRobot(pp.module.System):
```

Comment:
I think this is the more comprehensive name is the bicycle model. Could you add the name after you check the model is also the bicycle model?